bears/python/PycodestyleBear.py

```diff
@@ -61,3 +72,26 @@ def create_arguments(
         arguments.append(filename)
 
         return arguments
+
+    def process_output(
```

Comment:
all args fit on one line, and should be on one line.  The signature of this method does not change.